packages/manager/src/features/IAM/Users/UserDetailsLanding.tsx

```diff
@@ -71,10 +76,10 @@ export const UserDetailsLanding = () => {
             <p>user details - UIE-8137</p>
           </SafeTabPanel>
           <SafeTabPanel index={++idx}>
-            <p>UIE-8138 - User Roles - Assigned Roles Table</p>
+            <UserRoles assignedRoles={assignedRoles ?? {}} />
           </SafeTabPanel>
           <SafeTabPanel index={++idx}>
-            <p>UIE-8139 - User Roles - Resources Table</p>
+            <UserResources assignedRoles={assignedRoles ?? {}} />
```

Comment:
Is there any specific reason for using `{}` as a fallback? We could just pass 
```suggestion
            <UserResources assignedRoles={assignedRoles} />
```

And handle `undefined` rather than `{}`

Reply:
yep, updated it as well, thanks!